build.sc

```diff
@@ -2,7 +2,9 @@ import mill._
 import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 import scalalib._
 
-object requests extends Cross[RequestsModule]("2.12.6", "2.13.0")
+val dottyVersion = Option(sys.props("dottyVersion"))
+
+object requests extends Cross[RequestsModule]((List("2.12.6", "2.13.0") ++ dottyVersion): _*)
```

Comment:
Could you add Dotty 0.26.0-RC1 here, so that it is tested against dotty as well?

Reply:
Good idea

Reply:
We'll need to make sure to upstream the dotty build for geny too, otherwise this won't compile outside of the community build